docs/docs/image-tutorial.md

```diff
@@ -0,0 +1,271 @@
+        // This feature is untested for sites hosted on Wordpress.com
+        useACF: true,
+      },
+    },
+  ],
+}
+```
+
+### Installing plugins to help with images
+Now you’ll configure gatsby-source-filesystem to load the image directory, add a GraphQL query to a page, add an image to the page, and then view the result in the browser.
+First, you’ll need to install a few plugins and their dependencies:
+
+```shell
+`npm install --save gatsby-transformer-sharp`
+`npm install --save gatsby-plugin-sharp`
+`npm install --save gatsby-image`
+`npm install --save gatsby-source-filesystem` 
+```
+
+Place these plugins in your `gatsby-config.js` like this:
+
+```javascript{112-121}
+module.exports = {
+ siteMetadata: {
+   title: 'Gatsby Wordpress Tutorial',
+ },
+ plugins: [
+   // https://public-api.wordpress.com/wp/v2/sites/gatsbyjsexamplewordpress.wordpress.com/pages/
+   /*
+    * Gatsby's data processing layer begins with “source”
+    * plugins. Here the site sources its data from Wordpress.
+    */
+   {
+     resolve: `gatsby-source-wordpress`,
+     options: {
+       /*
+       * The base URL of the Wordpress site without the trailing slash and the protocol. This is required.
+       * Example : 'gatsbyjswpexample.wordpress.com' or 'www.example-site.com'
+       */
+       baseUrl: `dev-gatbsyjswp.pantheonsite.io`,
+       // The protocol. This can be http or https.
+       protocol: `http`,
+       // Indicates whether the site is hosted on wordpress.com.
+       // If false, then the asumption is made that the site is self hosted.
+       // If true, then the plugin will source its content on wordpress.com using the JSON REST API V2.
+       // If your site is hosted on wordpress.org, then set this to false.
+       hostingWPCOM: false,
+       // If useACF is true, then the source plugin will try to import the Wordpress ACF Plugin contents.
+       // This feature is untested for sites hosted on Wordpress.com
+       useACF: true,
+     },
+   },
+   'gatsby-plugin-react-helmet',
+   'gatsby-transformer-sharp',
+   'gatsby-plugin-sharp',
+   {
+     resolve: `gatsby-source-filesystem`,
+     options: {
+       name: `images`,
+       path: `${__dirname}/src/images/`,
+     },
+   },
+ ],
+}
+```
+
+### Creating GraphQL queries that pull in images from WordPress
+
+Now you are ready to create a GraphQL query to pull in some images from the WordPress site before we can view them in your browser. You will create a query that pulls in the images.
+
+Run:
+
+```shell
+`gatsby develop`
+```
+
+Open localhost:8000 and localhost:8000/__graphql. 
+
+Here’s an example of creating specific widths and heights for images:
+
+```jsx
+{
+  allWordpressPost {
+    edges {
+      node {
+        childWordPressAcfPostPhoto {
+          photo {
+            localFile {
+              childImageSharp {
+                # edit the maxWidth value to generate resized
```

Comment:
This line mentions `maxWidth` which isn't used in this query... how about:

`# Try editing the "width" and "height" values`